chat/src/main/java/net/md_5/bungee/api/Keybind.java

```diff
@@ -0,0 +1,86 @@
+package net.md_5.bungee.api;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+import net.md_5.bungee.api.chat.KeybindComponent;
+
+/**
+ * All possible keybind values used by {@link KeybindComponent}.
+ */
+@RequiredArgsConstructor
+public enum Keybind
```

Comment:
I'm very confused as to what you're trying to achieve with this enum.
The default keys don't look like they're used anywhere, so what's the point of including them? (ie why would a plugin need to know that key.sprint=LCTRL (which is not a string that can easily be parsed (same with MMOUSE etc)?
The CUSTOM enum makes no sense either (a caller will always just get CUSTOM with no useful info). Maybe you want to reconsider this design and just have stuff like:
KeyBinds { public static final String JUMP = "key.jump";}
So that plugins can use arbitrary strings across the board without having to deal with CUSTOM